bin/git-fork

```diff
@@ -55,6 +90,12 @@ if [ "$origin" = true ]; then
 else
     # clone forked repo into current dir
     git clone "${remote_prefix}${user}/${project}.git" "$project"
```

Comment:
Git clone will not allow one to clone over an already existing repo directory. 
````
0:0 ᐅ git clone git@github.com:hickey/git-extras.git git-extras
fatal: destination path 'git-extras' already exists and is not an empty directory.
````